packages/prisma/schema.prisma

```diff
@@ -149,6 +150,7 @@ model EventType {
   useEventTypeDestinationCalendarEmail    Boolean                   @default(false)
   aiPhoneCallConfig                       AIPhoneCallConfiguration?
   isRRWeightsEnabled                      Boolean                   @default(false)
+  descriptionTranslations                 EventTypeTranslation[]
```

Comment:
This should be renamed to just `translations` or `fieldTranslations` since this list could point to any type of translated field.

Reply:
Did it